torch/serialization.py

```diff
@@ -1317,10 +1374,24 @@ def restore_location(storage, location):
 
 
 class StorageType:
+    """Represent a storage type with a specific data type (dtype)."""
```

Comment:
Similarly ok to omit documentation for this class from this PR since it was not one of the pydocstyle errors